src/signalo/edge_cases/management/commands/populate_edge_cases.py

```diff
@@ -41,4 +46,11 @@ def handle(self, *args, **options):
         # Create data for DifferentSrid
         DifferentSrid.objects.create(geom=f"Point(2600000 1200000)")
 
+        # Create data for models whose permissions we are testing against
+        TestPermissionAllowAny.objects.create(geom=f"Point(2600000 1200000)")
+        TestPermissionDefaultPermissionsSettings.objects.create(
+            geom=f"Point(2600000 1200000)"
+        )
+        TestPermissionIsAdminUserModel.objects.create(geom=f"Point(2800000 1300000)")
```

Comment:
For some reason I cannot identify this doesn't create any instance, making the tests proposed by this PR fail.